### ISSUE TITLE
Theme editor controls are unlabeled for screen readers.

### ISSUE THREAD

**Issue by blindgoofball** [godotengine/godot#119511]
### Tested versions

* 4.6.2.stable

### System information

Godot v4.6.2.stable - Windows 11 (build 26200) - Multi-window, 1 monitor - OpenGL 3 (Compatibility) - NVIDIA GeForce RTX 4070 Laptop GPU (NVIDIA; 32.0.15.9571) - Intel(R) Core(TM) Ultra 9 185H (22 threads) - 63.44 GiB memory

### Issue description

In the UI theme editor in the bottom dock, there are several unlabeled controls when using a screen reader. These include:
* All override items
* The tabs for override types
* The edit field after the overrides list in the tab order
I used the NVDA screen reader, specifically NVDA 2026.1.

### Steps to reproduce

* Create a new project
* Create a new theme resource
* Select the theme tab in the bottom panel
* Add a new type to the theme
* Use tab while a screen reader is running to reach the overrides section and
* This entire section is unlabeled
Note: Depending on where you are when you start pressing tab, you may get stuck in the multiline field in the preview section and may need to use other methods such as a mouse or screen reader navigation commands to get out of this field.

### Minimal reproduction project (MRP)

N/A

**Comment by puneetdixit200** [comment]
Hello maintainers, I would like to work on this issue if it is available. My plan is to audit the Theme Editor controls for missing accessible names, add the appropriate labels/descriptions, and verify the result with screen-reader-accessible metadata. Please assign this to me if that approach is acceptable.

**Comment by Zireael07** [comment]
@puneetdixit200 Go ahead and just work on it. Godot engine does not practice assigning issues to people

**Comment by AThousandShips** [comment]
Please be aware of our [guidelines](https://contributing.godotengine.org/en/latest/pull_requests/pull_request_guidelines.html#ai-assisted-contributions) though

**Comment by bruvzg** [comment]
> Depending on where you are when you start pressing tab, you may get stuck in the multiline field in the preview section and may need to use other methods such as a mouse or screen reader navigation commands to get out of this field.

You can switch `tab` behavior in the multiline fields between adding tab character and switching focus by pressing `ui_focus_mode` action (`Ctrl + M` by default).
